docs/notebooks/03_merge.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>I think <code>skipinitialspace=True</code>.That bash script is too much magic, I wouldn't dare to touch it. </p><p> </p><p>Alternative is to use <code>gtfparse</code> in python directly, see an example here: </p><p>https://github.com/icbi-lab/infercnvpy/blob/4ac5e65fa54b1e5d5499a5305e80da7db50c6520/src/infercnvpy/io/_genepos.py#L41-L43</p><p> </p><p>STAR protocol papers should also have a "troubleshooting" section:</p><blockquote><em>Describe any problems that might arise from running the protocol and provide possible solutions. Make sure to flag them in the corresponding steps of the protocol and refer to this section. If applicable, this section can also be the space to include information about alternates, i.e., what reagents and/or equipment has some flexibility and what cannot be changed. If you wish to include a list for each problem/potential solution, please </em><strong><em>use bullet points</em></strong><em> (up to three level) make a list. </em><strong><em>Do not use numbering</em></strong><em> (1,2,3; a,b,c; i,ii,iii) in this section.</em></blockquote><p> </p><p>It could be an option to put alternative approaches there (we can make a separate notebook to have the same structure as the paper). </p>